docker-database/docker-compose.yml

```diff
@@ -0,0 +1,16 @@
+version: '3.8'
```

Comment:
```suggestion
```
- the version 3.x of compose syntax is only for swarm. 
- but good news: the attribute « version » is deprecated so you can remove it and forget it forever

Reply:
removed